lib/install/management/virtual_app.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/vmware/vic/lib/install/data"
 	"github.com/vmware/vic/pkg/errors"
 	"github.com/vmware/vic/pkg/trace"
+  "github.com/vmware/vic/pkg/version"
```

Comment:
yeah i was wondering why spacemacs was not allowing me to do indentation the way i wanted. another new stuff learned